app/components/form_components/autocomplete/script.js

```diff
@@ -44,6 +46,10 @@ const setupAutoComplete = (component) => {
     templates: { suggestion: (value) => suggestion(value, options) },
     name: rawFieldName
   })
+
+  if (in_error) {
+    component.querySelector('input').value = input_value
```

Comment:
Should the selector be more specific here as to which one it's changing the value for? Or is it within the context of the wrapping element for the autocomplete?

Reply:
It's in the context of the wrapping element for the autocomplete - there will only be one input element.